packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSearchBar/SearchBarDropdown.tsx

```diff
@@ -100,10 +100,15 @@ export const SearchBarDropdown: FC<SearchBarDropdownProps> = ({
             />
           ))}
         </div>
-      ) : (
+      )}
+      {!showResults && searchTerm.length >= 3 ? (
         <div className="no-results">
           <CWText type="b2">No results found</CWText>
         </div>
+      ) : (
+        <div className="no-results">
+          <CWText type="b2">Search term requires 3 or more characters</CWText>
+        </div>
```

Comment:
this is displayed even when there are 3 chars

<img width="631" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/0ae1ccdf-499e-4875-98db-651398859008">